app/services/archives_space/client.rb

```diff
@@ -41,10 +41,9 @@ def become(other)
       username = other.respond_to?(:aspace_username) ? other.aspace_username : other.to_s
       self.became_auth = post("/users/#{username}/become-user", nil)
       if block_given?
+        # yields to block, "unbecomes" user, and returns value of block
         yield(self).tap { unbecome }
       end
-    rescue Net::HTTPServerException => e
```

Comment:
This was removed b/c will never happen. The `post` method already catches `Net::HTTPServerException` and raises `ArchivesSpace::Client::Error`.